python_files/supplementary_plot_helpers.py

```diff
@@ -406,3 +414,101 @@ def stitch_before_after_norm(
 
     pre_norm_tiled.save(pre_norm_stitched_path)
     post_norm_tiled.save(post_norm_stitched_path)
+
+
+def euclidean_timepoint(tp_one_data: pd.Series, tp_two_data: pd.Series) -> float:
```

Comment:
This doesn't really line up with the original implementation. Is there a reason you didn't follow that logic?

https://github.com/angelolab/TNBC_python_scripts/blob/main/python_files/tumor_evolution/tumor_evolution.py#L13